stock_available_base_exclude_location/tests/common.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+import mock
+from operator import attrgetter
+from odoo import models
+
+
+class TestExcludeLocationMixin(object):
+    # Generate xmlids
+    # This is needed if you want to load data tied to a test model via xid.
+    _test_setup_gen_xid = False
+    # If you extend a real model (ie: res.partner) you must enable this
+    # to not delete the model on tear down.
+    _test_teardown_no_delete = False
+    # You can add custom fields to real models (eg: res.partner).
+    # In this case you must delete them to leave registry and model clean.
+    # This is mandatory for relational fields that link a fake model.
+    _test_purge_fields = []
+
+    @classmethod
+    def _test_setup_model(cls, env):
+        """Initialize it."""
+        with mock.patch.object(env.cr, "commit"):
+            cls._build_model(env.registry, env.cr)
+            env.registry.setup_models(env.cr)
+            ctx = dict(env.context, update_custom_fields=True)
+            env.registry.init_models(env.cr, [cls._name], ctx)
+
+    @classmethod
+    def _test_teardown_model(cls, env):
```

Comment:
@rousseldenis I've in mind that @sebastienbeau created a python lib providing this kind of functionality. (used in connector_search_engine)

Reply:
I think you are talking about this lib https://github.com/akretion/odoo-test-helper
I hope it can end up adopted in the OCA.

Reply:
> I think you are talking about this lib https://github.com/akretion/odoo-test-helper
> I hope it can end up adopted in the OCA.

Great!!! :rocket: